docs/Caching.md

```diff
@@ -0,0 +1,158 @@
+# Caching
+
+Web platform apps cache data at a few different levels in order to improve UI
+responsiveness. There are two primary types of content that are cached:
+
+1. REST API responses
+2. Static assets (files)
+
+## Overview
+
+![Cache diagram](https://user-images.githubusercontent.com/1885153/27892985-d75546fe-6256-11e7-8272-66252caa5c85.png)
```

Comment:
I'll update/clarify. Technically, all data _does_ go through the service worker, but the service worker passes through un-cached data transparently. The IndexedDB cache appears at the Redux middleware layer, not the 'browser' as a whole, so I'll make that more explicit